includes/options/class-amp-options-manager.php

```diff
@@ -17,20 +17,32 @@ class AMP_Options_Manager {
 	 */
 	const OPTION_NAME = 'amp-options';
 
+	/**
+	 * Slug for website experience.
+	 */
+	const WEBSITE_EXPERIENCE = 'website';
+
+	/**
+	 * Slug for stories experience.
+	 *
+	 * @var
+	 */
+	const STORIES_EXPERIENCE = 'stories';
+
 	/**
 	 * Default option values.
 	 *
 	 * @var array
 	 */
 	protected static $defaults = array(
+		'experiences'              => array( self::WEBSITE_EXPERIENCE ),
```

Comment:
Should we enable both experiences by default?

Reply:
Good question. Not sure. Part of me says no because historically it would be adding something that people didn't ask for. Jetpack's custom content types are disabled by default. 

I'd say leave it off by default now, and then we need to revamp the `readme.txt` to indicate that Stories is a key feature alongside Websites. (That's something that should be done prior to the 1.2 release.)

Then in 1.3 we can make it enabled by default, especially when Gutenberg is no longer a dependency, and when the Stories editor is more mature.

Reply:
Agree. Makes perfect sense. Let's follow that approach.

Reply:
Created #2567 for updating the readme.